src/components/Section/ProposalVoting/DelegationsLabel.tsx

```diff
@@ -2,25 +2,69 @@ import React from 'react'
 
 import Markdown from 'decentraland-gatsby/dist/components/Text/Markdown'
 import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { Popup } from 'decentraland-ui/dist/components/Popup/Popup'
+
+import Info from '../../Icon/Info'
+import VotingPower from '../../Token/VotingPower'
+import Username from '../../User/Username'
 
 import './DelegationsLabel.css'
 
 export interface DelegationsLabelProps {
   delegateLabel?: { id: string; values?: any }
   delegatorsLabel?: { id: string; values?: any }
+  infoMessage?: { id: string; values?: any }
+}
+
+function formatInfoMessage(infoMessage: { id: string; values?: any } | undefined) {
+  if (infoMessage && infoMessage.values) {
+    if (infoMessage.values.delegate) {
+      infoMessage.values.delegate = (
+        <Username className="DelegationsLabel__InfoMessage" address={infoMessage.values.delegate} addressOnly />
+      )
+    }
+    if (infoMessage.values.own_vp) {
+      infoMessage.values.own_vp = (
+        <VotingPower className="DelegationsLabel__InfoMessage" value={infoMessage.values.own_vp} />
+      )
+    }
+    if (infoMessage.values.delegated_vp) {
+      infoMessage.values.delegated_vp = (
+        <VotingPower className="DelegationsLabel__InfoMessage" value={infoMessage.values.delegated_vp} />
+      )
+    }
+  }
 }
 
-const DelegationsLabel = ({ delegateLabel, delegatorsLabel }: DelegationsLabelProps) => {
+const DelegationsLabel = ({ delegateLabel, delegatorsLabel, infoMessage }: DelegationsLabelProps) => {
   const t = useFormatMessage()
+  formatInfoMessage(infoMessage)
```

Comment:
I'd avoid reassigning prop as it gets confusing... Maybe this function can return a component that you can use in the Popup?